src/Panel/File.php

```diff
@@ -266,6 +266,40 @@ protected function imageSource(
 		return parent::imageSource($query);
 	}
 
+	/**
+	 * Whether focus can be added in Panel view
+	 */
+	public function isFocusable(): bool
+	{
+		// blueprint option
+		$option     = $this->model->blueprint()->focus();
+		// fallback to whether the file is viewable
+		// (images should be focusable by default, others not)
+		$option   ??= $this->model->isViewable();
+
+		if ($option === false) {
+			return false;
+		}
+
+		// ensure that user can update content file
+		if ($this->options()['update'] === false) {
+			return false;
+		}
+
+		$kirby = $this->model->kirby();
+
+		// ensure focus is only added when editing primary/only language
+		if (
```

Comment:
I think this is the wrong place. A file is still focusable even if a non-default language is selected. I think we need to make sure that the focus value is handled differently in writes and only added to the default language there instead.

Reply:
But `isFocusable` means whether a focus can be set - and a focus should only be set in the primary language. It can be read in any. Keep in mind that this is the File-Panel class not the main File class.

Reply:
Oh, I missed that it's not in the File class. Then it totally makes sense.